babel/messages/pofile.py

```diff
@@ -577,10 +585,8 @@ def _write_message(message, prefix=''):
                 comment_header = u'\n'.join(lines)
             _write(comment_header + u'\n')
 
-        for comment in message.user_comments:
-            _write_comment(comment)
-        for comment in message.auto_comments:
-            _write_comment(comment, prefix='.')
+        _write_comment(message.user_comments)
+        _write_comment(message.auto_comments)
```

Comment:
Why do we no longer use the `'.'` prefix here?

Reply:
Thank you for this observation. I will be making a new commit for the corrected code.